### DISCUSSION TITLE
Scheduling Posts

### DISCUSSION THREAD

**Discussion by cmgorton** [forem/forem#15473] [Completed]
[Originally posted](https://forem.dev/ioscasey/feature-request-scheduling-posts-1lag) by @FounderCasey on forem.dev 
(Also requested [here](https://forem.dev/cpluta/scheduled-posts-3kgc))
User would like to be able to schedule a post by selecting a time and date for said post to go live

This could be added as a button, "Schedule Post" or added in the "post options"
![schedule post option](https://user-images.githubusercontent.com/21039864/143056405-ca02ca46-850e-483e-931b-51f686b34f8a.png)

**Comment by JennieOcken** [comment]
This request is being addressed with this issue: https://github.com/forem/forem/issues/15858

**Comment by thomasbnt** [comment]
:tada: https://dev.to/devteam/this-post-is-scheduled-for-1130am-299k
